### PR TITLE
coastal WFP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/DOFSTK/CGINS-DOFSTK-02499__20231116.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2499,CC_frequency_offset,-8.343085e+002,source_file DOFST-K_SBE43F_SN_2499_Calibration_2023-11-16.zip > 2499.cal and SBE 43F O2499 22Nov23-SOC.pdf
+43-2499,CC_oxygen_signal_slope,3.6260e-004,Soc adj from pdf
+43-2499,CC_residual_temperature_correction_factor_a,-4.664006e-003,
+43-2499,CC_residual_temperature_correction_factor_b,1.782318e-004,
+43-2499,CC_residual_temperature_correction_factor_c,-2.712166e-006,
+43-2499,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02499__20231116.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20231021.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2724,CC_frequency_offset,-8.117512e+002, source_file DOFST-K_SBE43F_SN_2724_Calibration_2023-10-21.zip > 2724.cal and SBE 43F O2724 21Oct23-SOC.pdf
+43-2724,CC_oxygen_signal_slope,3.0610e-004,Soc adj from pdf
+43-2724,CC_residual_temperature_correction_factor_a,-5.717805e-003,
+43-2724,CC_residual_temperature_correction_factor_b,2.235659e-004,
+43-2724,CC_residual_temperature_correction_factor_c,-3.173341e-006,
+43-2724,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20231021.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20231003.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2725,CC_frequency_offset,-8.241657e+002, source_file DOFST-K_SBE43F_SN_2725_Calibration_2023-10-03.zip > 2725.cal and SBE 43F O2725 29Nov23-SOC.pdf
+43-2725,CC_oxygen_signal_slope,3.0101e-004,Soc adj from pdf
+43-2725,CC_residual_temperature_correction_factor_a,-4.642951e-003,
+43-2725,CC_residual_temperature_correction_factor_b,1.802736e-004,
+43-2725,CC_residual_temperature_correction_factor_c,-2.566140e-006,
+43-2725,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20231003.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20240124.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2726,CC_frequency_offset,-8.008544e+002, source_file DOFST-K_SBE43F_SN_2726_Calibration_2024-01-24.zip > 2726.cal and SBE 43F O2726 14Feb24.pdf
+43-2726,CC_oxygen_signal_slope,2.2031e-004, Soc adj from pdf
+43-2726,CC_residual_temperature_correction_factor_a,-3.417831e-003,
+43-2726,CC_residual_temperature_correction_factor_b,1.678301e-004,
+43-2726,CC_residual_temperature_correction_factor_c,-2.734189e-006,
+43-2726,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20240124.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20231116.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3370,CC_frequency_offset,-8.091549e+002,source_file DOFST-K_SBE43F_SN_3370_Calibration_2023-11-16.zip > 3370.cal and SBE 43F O3370 28Nov23-SOC.pdf
+43-3370,CC_oxygen_signal_slope,3.2201e-004,Soc adj from pdf
+43-3370,CC_residual_temperature_correction_factor_a,-5.340391e-003,
+43-3370,CC_residual_temperature_correction_factor_b,2.407394e-004,
+43-3370,CC_residual_temperature_correction_factor_c,-3.695631e-006,
+43-3370,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20231116.csv:7]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01031__20230912.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+1031,CC_angular_resolution,1.076,Constant; source_file FLORT-K_BBFL2_SN_1031_Calibration_2023-09-12.zip > BBFL2-1031.dev
+1031,CC_dark_counts_cdom,49.32352941,
+1031,CC_dark_counts_chlorophyll_a,51.10294118,
+1031,CC_dark_counts_volume_scatter,50.79411765,
+1031,CC_depolarization_ratio,0.039,Constant
+1031,CC_measurement_wavelength,700,Constant
+1031,CC_scale_factor_cdom,9.056E-02,
+1031,CC_scale_factor_chlorophyll_a,1.210E-02,
+1031,CC_scale_factor_volume_scatter,3.024E-06,
+1031,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01031__20230912.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01100__20230913.csv [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+1100,CC_angular_resolution,1.076,Constant; source_file FLORT-K_BBFL2_SN_1100_Calibration_2023-09-13.zip > BBFL2-1100.dev
+1100,CC_dark_counts_cdom,50.25,
+1100,CC_dark_counts_chlorophyll_a,50.6,
+1100,CC_dark_counts_volume_scatter,50.575,
+1100,CC_depolarization_ratio,0.039,Constant
+1100,CC_measurement_wavelength,700,Constant
+1100,CC_scale_factor_cdom,9.025E-02,
+1100,CC_scale_factor_chlorophyll_a,1.217E-02,
+1100,CC_scale_factor_volume_scatter,3.026E-06,
+1100,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01100__20230913.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01118__20230922.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+1118,CC_angular_resolution,1.076,Constant; source_file FLORT-K_BBFL2_SN_1118_Calibration_2023-09-22.zip > BBFL2-1118.dev
+1118,CC_dark_counts_cdom,48,
+1118,CC_dark_counts_chlorophyll_a,49,
+1118,CC_dark_counts_volume_scatter,47.525,
+1118,CC_depolarization_ratio,0.039,Constant
+1118,CC_measurement_wavelength,700,Constant
+1118,CC_scale_factor_cdom,9.060E-02,
+1118,CC_scale_factor_chlorophyll_a,1.198E-02,
+1118,CC_scale_factor_volume_scatter,2.957E-06,
+1118,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01118__20230922.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01119__20230922.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+1119,CC_angular_resolution,1.076,Constant; source_file FLORT-K_BBFL2_SN_1119_Calibration_2023-09-22.zip > BBFL2-1119.dev
+1119,CC_dark_counts_cdom,47,
+1119,CC_dark_counts_chlorophyll_a,49,
+1119,CC_dark_counts_volume_scatter,47.775,
+1119,CC_depolarization_ratio,0.039,Constant
+1119,CC_measurement_wavelength,700,Constant
+1119,CC_scale_factor_cdom,9.050E-02,
+1119,CC_scale_factor_chlorophyll_a,1.209E-02,
+1119,CC_scale_factor_volume_scatter,2.998E-06,
+1119,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01119__20230922.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01466__20230912.csv [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+1466,CC_angular_resolution,1.076,Constant; source_file FLORT-K_BBFL2_SN_1466_Calibration_2023-09-12.zip > BBFL2-1466.dev
+1466,CC_dark_counts_cdom,49.075,
+1466,CC_dark_counts_chlorophyll_a,54.15833333,
+1466,CC_dark_counts_volume_scatter,50.7,
+1466,CC_depolarization_ratio,0.039,Constant
+1466,CC_measurement_wavelength,700,Constant
+1466,CC_scale_factor_cdom,9.022E-02,
+1466,CC_scale_factor_chlorophyll_a,1.308E-02,
+1466,CC_scale_factor_volume_scatter,3.040E-06,
+1466,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01466__20230912.csv:11]: looks good

File: calibration/PARADK/CGINS-PARADK-20447__20231206.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.24,in mV; source_file PARAD-K_QSP2200_SN_20447_Calibration_2023-12-06.pdf
+20447,CC_scale_wet,9.75E-18,in Volts/(quanta/cm^2sec)
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20447__20231206.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20465__20231206.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20465,CC_dark_offset,1.05,in mV; source_file PARAD-K_QSP2200_SN_20465_Calibration_2023-12-06.pdf
+20465,CC_scale_wet,9.17E-18,in Volts/(quanta/cm^2sec)
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20465__20231206.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20468__20231206.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20468,CC_dark_offset,1.03,in mV; source_file PARAD-K_QSP2200_SN_20468_Calibration_2023-12-06.pdf
+20468,CC_scale_wet,9.22E-18,in Volts/(quanta/cm^2sec)
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20468__20231206.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20542__20231206.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20542,CC_dark_offset,1.19,in mV; source_file PARAD-K_QSP2200_SN_20542_Calibration_2023-12-06.pdf
+20542,CC_scale_wet,9.24E-18,in Volts/(quanta/cm^2sec)
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20542__20231206.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20582__20231206.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20582,CC_dark_offset,1.01,in mV; source_file PARAD-K_QSP2200_SN_20582_Calibration_2023-12-06.pdf
+20582,CC_scale_wet,9.21E-18,in Volts/(quanta/cm^2sec)
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20582__20231206.csv:3]: looks good

**Review by cdobs — APPROVED** [review]

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20240124.csv (hunk 4 above)
Comment (cooleyky) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20240124.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20231003.csv (hunk 3 above)
Comment (cooleyky) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20231003.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20231021.csv (hunk 2 above)
Comment (cooleyky) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20231021.csv:7]: looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02499__20231116.csv [hunk 16]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2499,CC_frequency_offset,-8.343085e+002,source_file DOFST-K_SBE43F_SN_2499_Calibration_2023-11-16.zip > 2499.cal and SBE 43F O2499 22Nov23 -SOC.pdf
+43-2499,CC_oxygen_signal_slope,3.6260e-004,Soc adj from pdf
+43-2499,CC_residual_temperature_correction_factor_a,-4.664006e-003,
+43-2499,CC_residual_temperature_correction_factor_b,1.782318e-004,
+43-2499,CC_residual_temperature_correction_factor_c,-2.712166e-006,
+43-2499,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cooleyky) [inline calibration/DOFSTK/CGINS-DOFSTK-02499__20231116.csv:7]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**